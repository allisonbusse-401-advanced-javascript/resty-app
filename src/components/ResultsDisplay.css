.ResultsDisplay {
  background-color: white;
  min-width: 900px;
  min-height: 500px;
  padding: 10px;
  margin-bottom: 50px;
}