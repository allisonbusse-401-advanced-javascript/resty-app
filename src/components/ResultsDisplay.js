import React from 'react';
import styles from './ResultsDisplay.css';
import PropTypes from 'prop-types';


const ResultsDisplay = ({ headers, response }) => (

  <div>
    <span>"Headers" : {headers}</span>
    <span>"Response" : {response}</span>
  </div>
);

ResultsDisplay.propTypes = {
  headers: PropTypes.string.isRequired,
  response: PropTypes.string.isRequired
};



export default ResultsDisplay;
