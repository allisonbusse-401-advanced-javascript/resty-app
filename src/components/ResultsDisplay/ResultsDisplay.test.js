import React from 'react';
import { shallow } from 'enzyme';
import ResultsDisplay from './ResultsDisplay';

describe('ResultsDisplay component', () => {
  const props = {
    headers: {},
    response: {},
  };

  it('renders ResultsDisplay', () => {   
    const wrapper = shallow(<ResultsDisplay {...props}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
