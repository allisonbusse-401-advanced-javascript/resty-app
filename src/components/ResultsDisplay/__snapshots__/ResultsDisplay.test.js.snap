// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsDisplay component renders ResultsDisplay 1`] = `
<div
  className="ResultsDisplay"
>
  <pre>
    "Headers" : 
    {}
  </pre>
  <pre>
    "Response" : 
    {}
  </pre>
</div>
`;
